### PR TITLE
Add KGE notebook to docs

### PR DESCRIPTION
**Pull request by nvitucci**

### CODE REVIEW

File: doc/modules/ROOT/pages/tutorials/kge-predict-transe-pyg-train.adoc [hunk 1]
```diff
@@ -0,0 +1,469 @@
...
+
+    torch.save(model, f"./model_{epoch_count}.pt")
+
+    return model
+----
+
+[source, python, role=no-test]
+----
+model = train_model_with_pyg()
+# The model can be loaded is it was trained before
```
Comment (orazve) [inline doc/modules/ROOT/pages/tutorials/kge-predict-transe-pyg-train.adoc:330]:
```suggestion
# The model can be loaded if it was trained before
```

Comment (orazve) [reply doc/modules/ROOT/pages/tutorials/kge-predict-transe-pyg-train.adoc:330]: I'll align this change with original notebook text.

**Review by orazve — APPROVED** [review]
LGTM and it repeats python notebook example.

File: doc/modules/ROOT/pages/tutorials/kge-predict-transe-pyg-train.adoc (hunk 1 above)
Comment (nvitucci) [reply doc/modules/ROOT/pages/tutorials/kge-predict-transe-pyg-train.adoc:330]: @orazve Well spotted! I can change it for you and recreate the Asciidoc.

**Pull request merged**